native/acter/src/api/timeline/message.rs

```diff
@@ -0,0 +1,986 @@
+use acter_core::models::status::{
```

Comment:
more tests needed. please look at this file in review mode to see which parts a not covered.